### PR TITLE
Documented some new 4.8 APIs

### PR DESCRIPTION
**Pull request by rpetrusha**

### CODE REVIEW

File: xml/System.Reflection/MethodImplAttributes.xml [hunk 1]
```diff
@@ -681,7 +683,7 @@
       </ReturnValue>
       <MemberValue>1024</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Specifies that the JIT compiler should look for security mitigation attributes, such as the user-defined <see cref="System.Runtime.CompilerServices.SecurityMitigationsAttribute" />. If found, the JIT compiler applies any related security mitigations. Available starting with .NET Framework 4.8.</summary>
```
Comment (mairaw) [inline xml/System.Reflection/MethodImplAttributes.xml]: This API doesn't seem to exist @rpetrusha and the right link would start with T:

File: xml/System.Threading/Interlocked.xml [hunk 2]
```diff
@@ -1559,8 +1559,14 @@ This method wraps a call to [FlushProcessWriteBuffers](/windows/desktop/api/proc
...
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Defines a memory fence that blocks speculative execution past this point until pending reads and writes are complete.</summary>
+        <remarks>  
+          <format type="text/markdown"><![CDATA[  
+
+The `SpeculationBarrier` method defines a memory fence (an `mfence` instruction on x86 and amd64 processors) that blocks *speculative* execution past this point until pending reads and writes are complete. 
+
+       ]]></format>
```
Comment (mairaw) [inline xml/System.Threading/Interlocked.xml]: I have a slight impression that too many spaces here cause an issue for loc but I could be wrong 

```suggestion
 ]]></format>
```

**Comment by mairaw** [comment]
Started a new build

**Comment by mairaw** [comment]
Good to merge @rpetrusha?

**Comment by rpetrusha** [comment]
Yes, @mairaw.

**Pull request merged**